### ISSUE TITLE
Overfit or at least looking like overfit while training ResNet

### ISSUE THREAD

**Issue by szseyd** [AINature/Classification-models#3]
While training the ResNet with dataset_lar_v2, the overfit issue occurred in validation accuracy and loss.

**Comment by szseyd** [comment]
Tuning the hyperparameters as increasing the batch size and augmenting the data helped but could not solve the problem completely.

**Comment by szseyd** [comment]
While the dataset_lar_v4 was fed to the network with more number of images,  the previously observed overfit tended to fix by a huge extent.
